ct/paymenter.sh

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env bash
+source <(curl -s https://raw.githubusercontent.com/community-scripts/ProxmoxVE/main/misc/build.func)
+
+# Copyright (c) 2021-2025 community-scripts ORG
+# Author: Nícolas Pastorello (opastorello)
+# License: MIT | https://github.com/community-scripts/ProxmoxVE/raw/main/LICENSE
+
+# App Default Values
+APP="Paymenter"
+var_tags="hosting;ecommerce;marketplace;"
+var_cpu="2"
+var_ram="1024"
+var_disk="5"
+var_os="Ubuntu"
+var_version="24.04"
```

Comment:
```suggestion
var_os="Debian"
var_version="12"
```
Why Ubuntu? Debian is standard.